LordUche/ireporter-react
# #165871739 Add Single Incident Page

#### What does this PR do?
Adds single incidents page
#### Description of Task to be completed?
- use google maps API to display location
- create view page
#### How should this be manually tested?
- ensure that build is passing on `Travis CI`
- check the review app to make sure everything works well on `Heroku`
#### Any background context you want to provide?
As a User, I should be able to view a single red-flag/intervention incident created by me
#### What are the relevant pivotal tracker stories?
[#165871739](https://www.pivotaltracker.com/story/show/165871739)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
